README.md

```diff
@@ -1 +1,58 @@
-# [WIP] Vividus extension for Visual Studio Code
\ No newline at end of file
+# Development guide
+
+## Tools
+* Node JS v12.13.0 or higher
+* Open JDK 12.0.1 or higher
+* Eclipse IDE 2019-09 R (4.13.0) or higher
+* Visual Studio Code 1.47.1 or higher
+
+## General
+1. Run `git clone https://github.com/vividus-framework/vividus-studio.git`
+2. Run `npm install -g yo generator-code`
+5. Install `Eclipse PDE (Plug-in Development Environment)` plugin in Eclipse IDE through `Eclipse Marketplace`
+
+## Import project to Eclipse
+### Import plugin
+1. Open `Eclipse IDE`
+2. Go to `File` tab in main menu and select `Import...`
+4. Select `Plug-ins and fragments` under `Plug-in development` section
+5. Set import directory to `<path-to-repo>/vividus-studio/vividus-studio-server` under `Import From` section
+6. Select `Select from all plug-ins and fragments found at the specified location` under `Plug-ins and Fragments to Import` section
+7. Select `Projects with source folders` under `Import As` section
+8. Click `Add all ->` and then `Finish` buttons
+9. Add the follwing entries into `<path-to-repo>/vividus-studio/vividus-studio-server/vividus-studio-plugin/.classpath` XML file under `<classpath>` section and then refresh the project
+```
+<classpathentry exported="true" kind="lib" path="external/slf4j-api.jar"/>
+<classpathentry exported="true" kind="lib" path="external/log4j-api.jar"/>
+<classpathentry exported="true" kind="lib" path="external/log4j-core.jar"/>
+<classpathentry exported="true" kind="lib" path="external/log4j-slf4j18-impl.jar"/>
+<classpathentry exported="true" kind="lib" path="external/junit-jupiter-api.jar"/>
+<classpathentry exported="true" kind="lib" path="external/junit-jupiter-params.jar"/>
+<classpathentry exported="true" kind="lib" path="external/mockito-core.jar"/>
+<classpathentry exported="true" kind="lib" path="external/mockito-junit-jupiter.jar"/>
+<classpathentry exported="true" kind="lib" path="external/hamcrest.jar"/>
+<classpathentry exported="true" kind="lib" path="external/opentest4j.jar"/>
```

Comment:
Is this the only thing that should be in `.classpath`?
Then we should provide it ready for copy-pasting:
```
<?xml version="1.0" encoding="UTF-8"?>
<classpath>
  <classpathentry exported="true" kind="lib" path="external/slf4j-api.jar"/>
  <classpathentry exported="true" kind="lib" path="external/log4j-api.jar"/>
  <classpathentry exported="true" kind="lib" path="external/log4j-core.jar"/>
  <classpathentry exported="true" kind="lib" path="external/log4j-slf4j18-impl.jar"/>
  <classpathentry exported="true" kind="lib" path="external/junit-jupiter-api.jar"/>
  <classpathentry exported="true" kind="lib" path="external/junit-jupiter-params.jar"/>
  <classpathentry exported="true" kind="lib" path="external/mockito-core.jar"/>
  <classpathentry exported="true" kind="lib" path="external/mockito-junit-jupiter.jar"/>
  <classpathentry exported="true" kind="lib" path="external/hamcrest.jar"/>
  <classpathentry exported="true" kind="lib" path="external/opentest4j.jar"/>
</classpath>
```